### DISCUSSION TITLE
Placeholders chars being replaced by those of another type by automatic translation

### DISCUSSION THREAD

**Discussion by sbocahu** [WeblateOrg/weblate#4878] [General]
Hi,

I just discovered Weblate and began testing it: it looks very promising, nice work !

It took me some time to figure it out but it seems that format/placeholders tags are not only used for quality checks but also to prevent placeholders to be translated during automatic translation, which is a killer feature I was looking for !

However, while some of my tests are giving exactly what I was looking for, some others don't:

It appears that some placeholders/format types character-delimiters are not respected and replaced by the angularjs/jinja2 one: {{ }}

ex: %var% replaced to {{var}}, or {var} replaced to {{var}}

Strangely, that's not 100% of my test strings...

Any explanation for this behaviour or should I fill a bug on github ?

Thanks,

Sebastien

**Comment by nijel** [comment]
Are the placeholders correctly detected in that case when rendering the string? I don't think Weblate would change the placeholder type, but there can always be a bug...

**Comment by sbocahu** [comment]
I could not reproduce recently. I think I got trapped by the "weblate memory" or something like that. I must had click on a suggestion
